project-base/storefront/hooks/app/usePageLoader.ts

```diff
@@ -25,12 +25,10 @@ export const usePageLoader = () => {
 
         router.events.on('routeChangeStart', onRouteChangeStart);
         router.events.on('routeChangeComplete', onRouteChangeStop);
-        router.events.on('routeChangeError', onRouteChangeStop);
 
         return () => {
             router.events.off('routeChangeStart', onRouteChangeStart);
             router.events.off('routeChangeComplete', onRouteChangeStop);
-            router.events.off('routeChangeError', onRouteChangeStop);
```

Comment:
I am wondering. We do not want to handle this event at all? What will happen in case of router error? 🤔

Reply:
There is a lot of router errors even currently. They are not "errors" per se. It is triggered even when a route navigation is cancelled (router event while other is happening) or similar. What it caused was the skeleton to behave weirdly. Without it, I did not spot any errors. If we do spot any in the future, we can handle them then, okay?

Reply:
Yeah, sure. If it's tested then absolutely okay.